lib/helpers/__tests__/url-utils.test.js

```diff
@@ -71,6 +71,24 @@ describe('appendQueryObject', () => {
     ).toBe('/foo/bar?test=one%2Ctwo');
   });
 
+  test('preserves array for setfilter', () => {
```

Comment:
I think we'll want to keep this test to confirm that setfilter maintains its array shape in the url since the default condition is to turn it into a comma-delineated list

Reply:
I was mixed up in my previous comment, you're correct. Removed this test.